<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>445</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Programa de Pontos</string>
  </property>
  <property name="windowIcon">
   <iconset resource="arquivo_de_recursos.qrc">
    <normaloff>:/Imagens/Ejelbg.png</normaloff>:/Imagens/Ejelbg.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">rgb(0, 0, 86)</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="rb_saida">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Saída</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_entrada">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>70</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Entrada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_usuario">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>207</y>
      <width>229</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuário</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box_senha">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>284</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_senha">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>261</y>
      <width>229</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="box_usuario">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="arquivo_de_recursos.qrc">:/Imagens/Ejelbg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>logo</zorder>
   <zorder>rb_saida</zorder>
   <zorder>rb_entrada</zorder>
   <zorder>ok</zorder>
   <zorder>cancelar</zorder>
   <zorder>label_usuario</zorder>
   <zorder>box_senha</zorder>
   <zorder>label_senha</zorder>
   <zorder>box_usuario</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivos">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <addaction name="actionHorarios"/>
    <addaction name="actionDetalhes"/>
   </widget>
   <widget class="QMenu" name="menuVersao">
    <property name="title">
     <string>Versão</string>
    </property>
    <addaction name="actionQT_Creator"/>
   </widget>
   <widget class="QMenu" name="menuOpcoes">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionSuperUsuario"/>
    <addaction name="actionCadastro_de_membros"/>
   </widget>
   <addaction name="menuArquivos"/>
   <addaction name="menuOpcoes"/>
   <addaction name="menuVersao"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionHorarios">
   <property name="text">
    <string>Horarios</string>
   </property>
  </action>
  <action name="actionDetalhes">
   <property name="text">
    <string>Acessos</string>
   </property>
  </action>
  <action name="actionQT_Creator">
   <property name="text">
    <string>QT Creator</string>
   </property>
  </action>
  <action name="actionSuperUsuario">
   <property name="text">
    <string>SuperUsuario</string>
   </property>
  </action>
  <action name="actionCadastro_de_membros">
   <property name="text">
    <string>Cadastro de membros</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>box_usuario</tabstop>
  <tabstop>box_senha</tabstop>
  <tabstop>rb_entrada</tabstop>
  <tabstop>rb_saida</tabstop>
  <tabstop>ok</tabstop>
  <tabstop>cancelar</tabstop>
 </tabstops>
 <resources>
  <include location="arquivo_de_recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelar</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
